docs/src/pages/styles/basics/basics.md

```diff
@@ -212,7 +212,7 @@ const useStyles = makeStyles((theme) => ({
 
 ## Using the theme context
 
-Starting from v5, Material-UI is no longer using JSS as its default styling solution. If you still want to use the utilities exported by `@material-ui/styles`, you will need to provide the `theme` as part of the context. For this, you may use the `ThemeProvider` component available in `@material-ui/styles`, or if you are already using `@material-ui/core`, you can use the one exported in `@material-ui/core/styles`.
+Starting from v5, Material-UI is no longer using JSS as its default styling solution. If you still want to use the utilities exported by `@material-ui/styles`, you will need to provide the `theme` as part of the context. For this, you may use the `ThemeProvider` component available in `@material-ui/styles`, or if you are already using `@material-ui/core`, you should use the one exported in `@material-ui/core/styles`, so that the same `theme` can be available in the core components as well.
```

Comment:
```suggestion
Starting from v5, Material-UI is no longer using JSS as its default styling solution. If you still want to use the utilities exported by `@material-ui/styles`, you will need to provide the `theme` as part of the context. For this, you may use the `ThemeProvider` component available in `@material-ui/styles`, or if you are already using `@material-ui/core`, you should use the one exported from `@material-ui/core/styles`, so that the same `theme` can be available in the core components as well.
```

Also: Same as above regarding "core components"

Reply:
```suggestion
Starting from v5, Material-UI is no longer using JSS as its default styling solution. If you still want to use the utilities exported by `@material-ui/styles`, you will need to provide the `theme` as part of the context. For this, you may use the `ThemeProvider` component available in `@material-ui/styles`, or if you are already using `@material-ui/core`, you should use the one exported from `@material-ui/core/styles`, so that the same `theme` can be available for the components coming from '@material-ui/core'.
```

Is it more clear like this?